wrappers/core18_test.go

```diff
@@ -218,14 +222,15 @@ WantedBy=snapd.service
 		{"show", "--property=ActiveState", "snapd.snap-repair.timer"},
 		{"start", "snapd.snap-repair.timer"},
 		{"is-active", "snapd.socket"},
-		{"start", "--no-block", "snapd.service"},
-		{"start", "--no-block", "snapd.seeded.service"},
-		{"start", "--no-block", "snapd.autoimport.service"},
 		{"--user", "--global", "--no-reload", "disable", "snapd.session-agent.service"},
 		{"--user", "--global", "--no-reload", "enable", "snapd.session-agent.service"},
 		{"--user", "--global", "--no-reload", "disable", "snapd.session-agent.socket"},
 		{"--user", "--global", "--no-reload", "enable", "snapd.session-agent.socket"},
 		{"--user", "daemon-reload"},
+		{"start", "--no-block", "snapd.apparmor.service"},
+		{"start", "--no-block", "snapd.service"},
+		{"start", "--no-block", "snapd.seeded.service"},
+		{"start", "--no-block", "snapd.autoimport.service"},
```

Comment:
it seems this was the case already on master but it seems we start autoimport twice? a bit curious why

Reply:
The previous is strange, since we have `X-Snapd-Snap: do-not-start` in the unit file. I will have to investigate what does that. But I think it is wrong.

Reply:
Mocked autoimport in 
https://github.com/snapcore/snapd/blob/68fe7eb9f7496ec54f2b3373a5c8d1993b9eb822/wrappers/core18_test.go#L56
 does not have the `do-not-start` tag

Reply:
Ah! Well spotted!